### ISSUE TITLE
browser track for Lantermann UTR/ncRNA dataset

### ISSUE THREAD

**Issue by mah11** [pombase/website#62]
Data in spreadsheet attached to Jira PB-2623. Kim may have converted it to GFF at some point already; if not, that's to do.

Datatype: "Curated transcripts"
Method: Manually curated from PMID: 18641648 transcriptome data-
Strain: h90 WT
PMID:20118936

**Comment by ValWood** [comment]
Do we still have access to this file ?

**Comment by ValWood** [comment]
moved to
https://github.com/pombase/website/issues/1441

**Comment by mah11** [comment]
It's GSE16040 in GEO

**Comment by ValWood** [comment]
I'm looking for the curated transcripts file that was converted to GFF?

**Comment by kimrutherford** [comment]
> I'm looking for the curated transcripts file that was converted to GFF?

We have some files that used to be in Dropbox and are now archived on the server.  Do these look right?:

Lantermann_dutrow_utrs_chr1.embl
Lanterman_utrs_chr2.embl
Lanterman_utrs_chr3.embl
original_1.embl
original_2.embl
original_3.embl
utrs-Chr1.embl
utrs-Chr2.embl
utrs-Chr3.embl

I don't remember converting these to GFF and I can't find any corresponding GFF files.

/data/pombase/archive_from_dropbox/Data/Broad/utrs/

**Comment by ValWood** [comment]
I have a feeling these might be the files we used to do
https://www.pombase.org/faq/how-do-you-determine-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites

I will try to figure out the difference between the 3 sets in Artemis....
I will also check the format in the paper.
This isn't the broad dataset though (as the directory label states) (maybe some of the embl files are the broad and  we are only interested here in the ones labelled "Lanterman"

I now remember that we had this data somewhere in a local directory , but I can't remember where that was...

**Comment by ValWood** [comment]
It might have been under svn,  or somewhere on the old Oliver 0. This also might have had the scripts for the file priorities so it might be useful to locate...

**Comment by ValWood** [comment]
I have been looking for this. @kimrutherford  could you have a look at some point. I think this dataset would be useful to align with our curation.

**Comment by kimrutherford** [comment]
All I can find the files mentioned in this comment: https://github.com/pombase/website/issues/62#issuecomment-671910129=

Do you have any more clues you can give me?

**Comment by Antonialock** [comment]
Isn’t there a dismiss of everything that was in jira? Maybe search it for pb-2623?

**Comment by ValWood** [comment]
I  haven't had access to Jira since Po-xit. I'll go back to the paper. There are a few papers which mention revised gen annotation sets that I need to dig out and read. 

@kim I will get onto this after the 9th. Assigned to me for now.

**Comment by kimrutherford** [comment]
> Isn’t there a download of everything that was in jira? Maybe search it for pb-2623?

Thanks Antonia!  You're right.  I had forgotten that we downloaded a copy of the Jira tickets.  I've put a copy in Dropbox/pombase/temp/ebi_jira_tickets_backup

They are HTML files.  They render OK if you open them in Firefox.  These files mention "Lanterman":

jira-2623.html
jira-2624.html
jira-2003.html
jira-1353.html
jira-1088.html

These files mention "Dutrow":

jira-2624.html
jira-1353.html
jira-1088.html
jira-1018.html

I've had a look but I can't see any filenames that would help.

**Comment by Antonialock** [comment]
2623 mentions a file called
nsmb.1741-S2.xls

"



*This is the other set of manually curated features.(will upload)Its a
apreadsheet so it will need converting to gff. I'll assign to Kim first
because we used this dataset in the current UTRS (with lower precedence
than the Rhind data). I wonder if Kim already did the conversion somewhere.
I looked in the UTR files in the drop box, and I could only find EMBL
files.*

*It will appear under "transciptome"*




*Datatype- "Curated transcripts"Method-Manually curated from PMID: 18641648
transcriptome data-Strain h90 WTcitation PMID: 20118936"*

**Comment by Antonialock** [comment]
you might be able to request your username, and then password, using your
email if required..
https://www.ebi.ac.uk/panda/jira/browse/PB-2623

I can log in but not see this issue - I think because my pombase account
got merged with my new ebi account when I started here & I lost
permissions.

**Comment by kimrutherford** [comment]
Thanks again Antonia.  You're good at this.  :-)

**Comment by kimrutherford** [comment]
Sorry, hit return too quick

> 2623 mentions a file called
> nsmb.1741-S2.xls

I think that must be S2 of: https://www.nature.com/articles/nsmb.1741

**Comment by ValWood** [comment]
I woud quite like to be abe to view this dataset in the New Year for the genome update paper. I'd like to compare to the curated transcripts

**Comment by kimrutherford** [comment]
OK, let's chat about this soon.  I have questions about the dataset.

**Comment by kimrutherford** [comment]
Hi @PCarme

Let's have a look at this dataset in January.  This has been planned for a long time.  :-)

**Comment by ValWood** [comment]
yep, I have suggested opening a new ticket, one or each dataset, using a template.
